cmd/nvidia-dra-plugin/device_state.go

```diff
@@ -100,12 +104,16 @@ func NewDeviceState(ctx context.Context, config *Config) (*DeviceState, error) {
 		cdi:               cdi,
 		tsManager:         tsManager,
 		mpsManager:        mpsManager,
+		vfioPciManager:    vfioPciManager,
 		allocatable:       allocatable,
 		config:            config,
 		nvdevlib:          nvdevlib,
 		checkpointManager: checkpointManager,
 	}
 
+	// Initialize the vfio-pci driver manager.
+	vfioPciManager.Init()
```

Comment:
Why do we init this here? Where are other managers initialised?

Reply:
tsManager/mpsManager don't have any "initializations". But checkpointManager is initialized underneath: L117-L131.

What's the concern here?